lms/djangoapps/teams/models.py

```diff
@@ -192,7 +197,6 @@ def create(
             description=description,
             country=country if country else '',
             language=language if language else '',
```

Comment:
Nit: let's get rid of the `if/else` for `topic_id`, `country`, and `language`, since the default values are now handled in the definition of the fields.

Reply:
It looks like assigning a `None` value to a field doesn't trigger the default settings, since it already has a value, albeit an empty one. I am going to address it by changing from `None` to `''` for default values of parameters so we can get rid of that check.